code-pushup.config.ts

```diff
@@ -35,30 +36,30 @@ const config = {
   }),
 
   plugins: [
-    fileSizePlugin({
-      directory: './dist/packages',
-      pattern: /\.js$/,
-      budget: 174_080, // 170 kB
-    }),
-
-    packageJsonPlugin({
-      directory: './dist/packages',
-      license: 'MIT',
-      type: 'module',
-    }),
+    // fileSizePlugin({
+    //   directory: './dist/packages',
+    //   pattern: /\.js$/,
+    //   budget: 174_080, // 170 kB
+    // }),
+    //
+    // packageJsonPlugin({
+    //   directory: './dist/packages',
+    //   license: 'MIT',
+    //   type: 'module',
+    // }),
   ],
 
-  categories: [
-    {
-      slug: 'custom-checks',
-      title: 'Custom checks',
-      refs: [
-        ...fileSizeRecommendedRefs,
-        packageJsonPerformanceGroupRef,
-        packageJsonDocumentationGroupRef,
-      ],
-    },
-  ],
+  // categories: [
+  //   {
+  //     slug: 'custom-checks',
+  //     title: 'Custom checks',
+  //     refs: [
+  //       ...fileSizeRecommendedRefs,
+  //       packageJsonPerformanceGroupRef,
+  //       packageJsonDocumentationGroupRef,
+  //     ],
+  //   },
+  // ],
```

Comment:
I'd be fine with just deleting these from the config, to be honest :grin: I don't think they serve much purpose anymore.